packages/cursorless-engine/src/processTargets/targets/UntypedTarget.ts

```diff
@@ -65,3 +63,23 @@ export class UntypedTarget extends BaseTarget<UntypedTargetParameters> {
     };
   }
 }
+export function createContinuousRangeUntypedTarget(
```

Comment:
How come this function needs to be here? I know it produces an untyped target, but I'm not sure I see it as a factory